src/kakarot/evm.cairo

```diff
@@ -144,32 +145,52 @@ namespace EVM {
     // @param amount The amount of gas the current operation requires.
     // @return EVM The pointer to the updated execution context.
     func charge_gas{range_check_ptr}(self: model.EVM*, amount: felt) -> model.EVM* {
-        let out_of_gas = is_le_felt(self.gas_left + 1, amount);
+        tempvar a = self.gas_left - amount;
```

Comment:
rename `a` to an explicit name

Reply:
This `a` is needed for the hint. I thought we could not modify hints due to being deployed on starknet and needing whitelisted hints. Did I misunderstood something here ?

Reply:
ah, you're right! When inlined it makes less sense but we can't do anything about it